<launch>

    <arg name="project" default="data_tools"/>
    <arg name="type" default="aloha"/>
    
    <!-- Parameters -->
    <rosparam file="$(find data_tools)/config/$(arg type)_data_params.yaml" command="load" />

    <arg name="episodeIndex" default="-1"/>
    <arg name="datasetDir" default="/home/agilex/data"/>
    <param name="episodeIndex" value="$(arg episodeIndex)"/>
    <param name="datasetDir" value="$(arg datasetDir)"/>

    <node pkg="$(arg project)" type="$(arg project)_pointCloudFilter" name="$(arg project)_pointCloudFilter" output="screen" required="true"/>

</launch>
